### PR TITLE
add brew & scoop setup

### LINKED ISSUE #1: [1] Setup Deployments for CLI tool
Brew - macOS
Scoop - windows
Snap? - linux

### PR DESCRIPTION
**Pull request by svix-frank**
Okay @tasn I lied, both the brew and scoop need to live in separate repos, I propose two repos: `homebrew-svix` and `scoop-svix`

The Homebrew flow can then look like this 
```sh
brew tap svixhq/svix
brew install svix
# or
brew install svixhq/svix/svix
```
and the scoop flow will look like this:
```sh
scoop bucket add svix https://github.com/svixhq/scoop-svix.git
scoop install svix
```

Additionally before I can merge this, I need to get setup with access to `secrets` for this repo as well as write access to the newly created repos.

Closes #1.

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -36,6 +36,27 @@ nfpms:
...
       - rpm
       - apk
+brews:
+  - tap:
+      owner: svix
+      name: homebrew-svix
+    commit_author:
+      name: svix-ci
+      email: support@svix.com
+    homepage: https://svix.com
```
Comment (tasn) [inline .goreleaser.yml:46]:
```suggestion
    homepage: https://www.svix.com
```

File: .goreleaser.yml [hunk 2]
```diff
@@ -36,6 +36,27 @@ nfpms:
...
+      bin.install "svix"
+    caveats: "Thanks for installing the Svix CLI! If this is your first time using the CLI, checkout our docs at https://docs.svix.com."
+scoop:
+  bucket:
+    owner: svix
+    name: scoop
+  commit_author:
+    name: svix-ci
+    email: support@stripe.com
+  homepage: https://svix.com
```
Comment (tasn) [inline .goreleaser.yml:58]:
```suggestion
  homepage: https://www.svix.com
```

File: .goreleaser.yml [hunk 3]
```diff
@@ -36,6 +36,27 @@ nfpms:
...
+    install: |
+      bin.install "svix"
+    caveats: "Thanks for installing the Svix CLI! If this is your first time using the CLI, checkout our docs at https://docs.svix.com."
+scoop:
+  bucket:
+    owner: svix
+    name: scoop
+  commit_author:
+    name: svix-ci
+    email: support@stripe.com
```
Comment (tasn) [inline .goreleaser.yml:57]:
```suggestion
    email: support@svix.com
```

File: .goreleaser.yml [hunk 4]
```diff
@@ -36,6 +36,27 @@ nfpms:
...
+      email: support@svix.com
+    homepage: https://svix.com
+    description: Svix CLI utility
+    install: |
+      bin.install "svix"
+    caveats: "Thanks for installing the Svix CLI! If this is your first time using the CLI, checkout our docs at https://docs.svix.com."
+scoop:
+  bucket:
+    owner: svix
+    name: scoop
```
Comment (tasn) [inline .goreleaser.yml:54]:
```suggestion
    name: scoop-svix
```
So it matches the homebrew one?

Comment (svix-frank) [reply .goreleaser.yml:54]: sure

**Review by tasn — APPROVED** [review]
I don't understand how this works. Shouldn't owner be svixhq? Does this even work? :P

**Pull request merged**